src/kicad/common.ts

```diff
@@ -33,6 +33,11 @@ export function unescape_string(str: string): string {
     return str;
 }
 
+/** Object with a unique ID (UUID or tstamp). */
+export interface UniqueIDObject {
+    get uuid_text(): string | undefined;
```

Comment:
any reason for `uuid_text` over `unique_id` or just `uuid`? 

In the same vein,  this interface/type should probably be something like `HasUUID`/`HasUniqueID` like `HasResolveTextVars` below.

Reply:
Wonderful, this looks great. :)